config-connector/solutions/projects/kpt/project-services/README.md

```diff
@@ -0,0 +1,42 @@
+Project Services
+==================================================
+
+# NAME
+
+  project-services
+
+# SYNOPSIS
+
+  Config Connector compatible YAML files to enable services on a project.
+
+## Requirements
+  Your service account must have roles/serviceusage.serviceUsageAdmin or roles/owner
+
+
+## Usage
+  First, set the project-id:
+
+  ```
+  kpt cfg set . project-id VALUE 
+  ```
+
+  Before applying a service, set the service name and the project-id annotation. For example, to enable Compute Engine:
```

Comment:
Could you add this link https://cloud.google.com/compute/docs/reference/rest/v1/ here for Compute Engine? Customer might be wondering - what is Compute Engine? why the service name of Compute Engine is compute.googleapis.com? etc.

Reply:
Done

Reply:
While looking for a good link for PubSub I found a better entry point for Compute Engine here: https://cloud.google.com/compute/docs
My inclination is to use this sort of link for both APIs.